test/fuzz.js

```diff
@@ -0,0 +1,71 @@
+const crypto = require('crypto')
+const { it, describe } = require('mocha')
+const assert = require('assert')
+
+const { DDWAF } = require('..')
+const rules = require('./rules.json')
+
+const blns = require('./blns/blns.json')
+
+const waf = new DDWAF(rules)
+
+const TIMEOUT = 20000
+
+const ENCODINGS_0 = [ // from https://github.com/nodejs/node/blob/master/lib/buffer.js
+  'utf8',
+  'ucs2',
+  'utf16le',
+  'latin1',
+  'ascii',
+  'base64',
+  'base64url',
+  'hex'
+]
+const ENCODINGS = []
+for (const encoding of ENCODINGS_0) {
+  try {
+    Buffer.from('hello', encoding)
+    ENCODINGS.push(encoding)
+  } catch (_) {
+  }
+}
+
+const test = function (entry, encoding = 'utf8') {
+  const context = waf.createContext()
+  const r1 = context.run({
+    atk: entry
+  }, TIMEOUT)
+  assert(!!r1)
```

Comment:
`!!` is redundant. From the docs: `Tests if value is truthy. It is equivalent to assert.equal(!!value, true, message).`